### PR TITLE
Update apps-win32-add.md

### PR DESCRIPTION
**Pull request by msbemba**
Added more information about the path that it cannot contain " or ,

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by PRMerger16** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/memdocs/issues/2751

File: memdocs/intune/apps/apps-win32-add.md [hunk 1]
```diff
@@ -186,7 +186,7 @@ On the **Detection rules** pane, configure the rules to detect the presence of t
         - **MSI product code**: Add a valid MSI product code for the app.
         - **MSI product version check**: Select **Yes** to verify the MSI product version in addition to the MSI product code.
     - **File**: Verify based on file or folder detection, date, version, or size.
-        - **Path**: Enter the full path of the folder that contains the file or folder to detect.
+        - **Path**: Enter the full path of the folder that contains the file or folder to detect. This should not include special characters like , or " .
```
Comment (JohanFreelancer9) [inline memdocs/intune/apps/apps-win32-add.md:189]:
```suggestion
        - **Path**: Enter the full path of the folder that contains the file or folder to detect. This should not include special characters such as **,** or **"**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by PRMerger12** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by Erikre — APPROVED** [review]

**Comment by Erikre** [comment]
#sign-off

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**